CHANGELOG.md

```diff
@@ -6,12 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
-- All the code since the dawn of comit-rs
-
-[Unreleased]: https://github.com/comit-network/comit-rs/compare/1625533e04119e8496b14d5e18786f150b4fce4d...HEAD
-
-- Update handling of config files
-
-[Changed]: For configuration files we now use the standard location as expected on each platform (OSX, Windows, Linux).
+- All the code since the dawn of comit-rs.
 
+### Changed
+- Move config files to standard location based on platform (OSX, Windows, Linux).
 
+[Unreleased]: https://github.com/comit-network/comit-rs/compare/1625533e04119e8496b14d5e18786f150b4fce4d...HEAD
```

Comment:
This was under the 'Added' heading before I touched it, is it supposed to be under the 'Changed' heading now?

Reply:
I think the idea is that these diffs go at the bottom of the file, see https://keepachangelog.com/en/1.0.0/.

Reply:
This is markdown syntax for link-aliases if I am not mistaken: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Oh, yeah, I didn't even notice what it was doing haha. Instead of inlining the link you use a reference.